common/workunit/referencedfilelist.cpp

```diff
@@ -543,14 +548,15 @@ class ReferencedFileIterator : implements IReferencedFileIterator, public CInter
     Owned<HashIterator> iter;
 };
 
-void ReferencedFileList::ensureFile(const char *ln, unsigned flags, const char *pkgid, bool noDfsResolution, const char *daliip, const char *srcCluster, const char *prefix)
+void ReferencedFileList::ensureFile(const char *ln, unsigned flags, const char *pkgid, bool noDfsResolution, StringArray& subfileNames, const char *daliip, const char *srcCluster, const char *prefix)
```

Comment:
Should probably be "const StringArray *subfileNames".  const because it isn't an out parameter, pointer so you can pass nullptr when you don't need to pass subfiles.